README.md

```diff
@@ -1,323 +1,299 @@
 
-      --loglevel 6
-      --chainGenerate
-      --chainLength 10
-      --chainOutputPath C:\\Ethereum\\OutputPath
-      --chainGenesis C:\\Ethereum\\Input\\Genesis.json
-      --chainBlockTime 30
+If you want to rebuild both, separate the names with a `,` as such: 
+```text
+--docker-nocache devp2p,go-ethereum_latest
+```
 
-## Iterating on bugfixes locally
+# Adding a Simulation
+There are two components to a simulation: 
+1. a **simulation program written in Go** using the `hivesim` test API to coordinate the execution of your desired test; and
+2. a **Dockerfile** to containerize both the simulation and the tests to be executed against client implementations.
 
-Often, after reproducing a failure with a hive run on your machine, you'll want to build a docker container with code changes and see if it fixes the issue. Enabling this sort of workflow will require a couple of steps:
+## Placement
+_(This section is relevant if you plan to merge your simulation upstream)_
 
-* Build your client's docker image locally with a different tag to the one in your client's dockerfile.
-* Edit the FROM field in the dockerfile for your client, found in `clients/[your_client_name_here]/dockerfile`, to point to your newly created image
-* Run hive again with the flag `--docker-nocache <regexp>`, where `<regexp>` is a regular expression matching your client's name, such as `besu`
+If the theme of the test suite can be grouped in one of the directories located in `simulators/`, please place the new simulation in that directory. Otherwise, if the simulation cannot be categorized with the current groupings, create a new directory in `simulators/` and name it according to the theme of the test suite.
 
+@TODO-----------------------------------------
+Overriding environmental variables that change client behaviors via HTTP parameters is easy to do in
+any HTTP client utility and/or library, but uploading files needed for chain initializations is much
+more complex, especially if multiple files are needed. As long as all clients run with the same set
+of init files, this is not an issue (they can be placed in the default locations). However if instances
+need to run with different initial chain setups, a simulator needs to be able to specify these per
+client. To avoid file uploads, `hive` solves this by defining a set of API variables that allow a
+simulator to specify the source paths to use for specific init files which will be extracted from the
+live container:
 
-# Running on Windows
+ * `HIVE_INIT_GENESIS` path to the genesis file to seed the client with (default = "/genesis.json")
+ * `HIVE_INIT_CHAIN` path to an initial blockchain to seed the client with (default = "/chain.rlp")
+ * `HIVE_INIT_BLOCKS` path to a folder of blocks to import after seeding (default = "/blocks/")
+ * `HIVE_INIT_KEYS` path to a folder of account keys to import after init (default = "/keys/")
```

Comment:
These variables are not used anymore. The files are now supplied using the multipart file-upload request to start a node.